microsite/data/plugins/bugsnag.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Bugsnag
+author: roadie.io
+authorUrl: https://sda.se/
```

Comment:
👀

Reply:
@dhenneke Thank you for catching that :) Sorry for the confusion